### PR TITLE
[4.0] Updating dependencies for PHP and JS

### PR DESCRIPTION
**Pull request by Hackwar**
This does what the title says.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,44 +30,44 @@
     "not ie < 11"
   ],
   "dependencies": {
-    "@claviska/jquery-minicolors": "^2.3.4",
-    "@webcomponents/webcomponentsjs": "^2.4.3",
+    "@claviska/jquery-minicolors": "^2.3.5",
+    "@webcomponents/webcomponentsjs": "^2.4.4",
     "accessibility": "^3.0.9",
     "awesomplete": "1.1.5",
     "bootstrap": "^4.5.0",
```
Comment (Quy) [inline package.json:37]: Upgrade to 4.5.2?

**Comment by brianteeman** [comment]
this PR is already out of date

**Comment by richard67** [comment]
@Hackwar Could you check and if necessary update for meanwhile never versions where possible?

And could you include an update of bootstrap 4 from the 4.5.0 we have now to the recent 4.5.2?

The latter would solve issue #30319 .

If you don't have time I can try to help with a PR to your branch, at least for the bootstrap stuff. Just ping me if necessary.

**Comment by richard67** [comment]
This PR is meanwhile out of date, but there is a new one: #30334 , which also includes bootstrap 4 update.

Closing in favour of #30334 . Sorry @Hackwar that your PR had to wait to long for tests that it became outdated.

**Pull request closed without merging**